src/assets/DatasetsUtils.ts

```diff
@@ -52,6 +52,7 @@ const titles: { [key: string]: string } = {
   'contentfilterprofiles-singular': 'Content Filter Profile',
   'contentfilterrules': 'Content Filter Rules',
   'contentfilterrules-singular': 'Content Filter Rule',
+  'cloudfunctions-singular': 'Cloud Functions',
```

Comment:
```suggestion
  'cloudfunctions': 'Cloud Functions',
  'cloudfunctions-singular': 'Cloud Function',
```

Reply:
one title for singular, one title for multiple (non singular)

Reply:
```suggestion
  'cloudfunctions-singular': 'Cloud Function',
```

`'cloudfunctions-singular': 'Cloud Functions',` - That's still not singular

Reply:
Thanks